### PR TITLE
Revamp `genezio` supercommand

### PR DESCRIPTION
**Pull request by costinsin**
## Type of change

-   [x] 🍕 New feature

## Description
This change revamps the `genezio` super command to be smarter and guide new users on how to use our CLI.

Adds a new project creation using our public templates.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by costinsin** [comment]
Current dependencies on/for this PR:
* `dev`
  * **PR #604** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/604?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #618** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/618?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #641** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/641?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/641?utm_source=stack-comment).

File: src/commands/superGenezio/superGenezio.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            }
+            case "local": {
+                process.env["CURRENT_COMMAND"] = "local";
+                const options: GenezioLocalOptions = {
+                    port: PORT_LOCAL_ENVIRONMENT,
+                    installDeps: true,
+                };
+
+                return await startLocalEnvironment(options).catch((error) => {
+                    GenezioTelemetry.sendEvent({
```
Comment (radu1122) [inline src/commands/superGenezio/superGenezio.ts:65]: i think await is needed here

File: src/commands/superGenezio/createNewProject.ts [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+            `Unfortunately, no ${category} templates could be found for this project type :(. We are working on adding more templates!`,
+        );
+    }
+
+    const { selectedLanguage }: { selectedLanguage: string } = await inquirer.prompt([
+        {
+            type: "list",
+            name: "selectedLanguage",
+            message:
+                colors.magenta("Select your desired ") +
```
Comment (radu1122) [inline src/commands/superGenezio/createNewProject.ts:290]: if there is only 1 choice, automatically select it without the prompt

**Review by vladiulianbogdan — APPROVED** [review]

**Comment by costinsin** [comment]
### Merge activity

* **Dec 21, 11:57 AM**: @@costinsin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/641).

**Pull request merged**